desktop/apps/miami_fair_week/index.js

```diff
@@ -37,13 +44,20 @@ class EditableMiamFairWeekPage extends JSONPage {
       next(error)
     }
   }
+
+  serializeAsQueryString(params) {
+    return Object.keys(params)
+      .filter(key => params.hasOwnProperty(key))
+      .map(key => `${encodeURIComponent(key)}=${encodeURIComponent(params[key])}`)
+      .join("&")
+  }
```

Comment:
@damassi do you know any other way to serialize `req.query` back to a query string? Basically what I'm trying to do here is the same as [this SO question](https://stackoverflow.com/questions/1714786/query-string-encoding-of-a-javascript-object).

Reply:
This has been dependable over the years and could swear that its already in the project, but I guess not? https://www.npmjs.com/package/query-string. Probably no harm in adding though.

Reply:
:+1: